packages/ses/NEWS.md

```diff
@@ -1,4 +1,6 @@
-User-visible changes in `ses`:
+<!-- markdownlint-disable MD041 -->
```

Comment:
What's this?

Reply:
it fixes the markdownlint warning

<img width="698" alt="Screenshot 2025-03-26 at 4 23 25 pm" src="https://github.com/user-attachments/assets/7923091f-cf94-47c3-99da-cfe87873ea7c" />

Reply:
we don't have makedownlint (or cSpell) running in CI (yet)